docs/learn/universal-profile/metadata/edit-profile.md

```diff
@@ -129,98 +78,68 @@ Use the JSON template file below, and follow **steps 1.1 and 1.2** to fill the b
 }
 ```
 
-### 1.1 - Add profile details (links, tags, etc.)
+- **Add more details to your profile:**
 
-Add more details about the Universal Profile for the entity's name, description, links, and tags.
+Add more details about the Universal Profile for the entity's name, description, links, and tags. The properties `links` and `tags` accept an array of objects or strings, so you can add as many as you need!
 
 Be as creative as you want to make your Universal Profile as unique as possible! :art:
 
-:::info Learn More
-The properties `links` and `tags` accept an array of objects or strings, so you can add as many as you need!
+- **Add a profile and background image:**
 
-For editing the properties `profileImage` and `backgroundImage`, see **Step 1.2** below :arrow_down:
+:::info Recommendation
+The JSON file for [`LSP3Profile`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-3-Profile-Metadata.md#lsp3profile) accepts an array of images so that you have pictures of different sizes and dimensions. This way, client interfaces can know which files to pick based on the container size in their interface.
 :::
 
-### 1.2 - Add a profile and background image
+:::note Notice
+In this guide, we will use [local IPFS Node provider](https://github.com/lukso-network/tools-data-providers?tab=readme-ov-file#apps-and-packages) from our [IPFS Provider Tool](https://github.com/lukso-network/tools-data-providers). [Other providers](https://github.com/lukso-network/tools-data-providers?tab=readme-ov-file#apps-and-packages) supported by this tool including Pinata, Infura, Cascade or Sense Protocol.
+:::
 
-:::caution
-Image sizes should be written as numbers, not as strings.
+```js
+import { createReadStream } from 'fs';
+import { IPFSHttpClientUploader } from '@lukso/data-provider-ipfs-http-client';
 
-The **max image width** allowed on [universalprofile.cloud](https://universalprofile.cloud) is: `profileImage <= 800px`, `backgroundImage <= 1800px`
-:::
+const provider = new IPFSHttpClientUploader('http://127.0.0.1:5001/api/v0/add');
 
-:::info
-The JSON file for LSP3Profile accepts an array of images so that you have pictures of different sizes and dimensions.<br/>
-This way, client interfaces can know which files to pick based on the container size in their interface.
-:::
+const file = createReadStream('./test-image.png');
+
+const { url, hash } = await provider.upload(file);
+```
 
 For the properties `profileImage` and `backgroundImage`, we will need to add the following information:
```

Comment:
I think this whole section should go above. The order seems off. As an extra paragraph here:

<img width="1006" alt="image" src="https://github.com/user-attachments/assets/ed89e0af-86e4-443f-a051-9c02881925ec">

An you can say for `url` as follow:

---
- `hash`: use this **[keccak256 image hash generator](https://emn178.github.io/online-tools/keccak_256_checksum.html)**.
- `url`: the url of the image uploaded to IPFS.

To obtain the `url`, let's go to the next section below to learn how to upload the profile images + the final JSON file to IPFS! Using our [`@lukso/data-provider-ipfs-http-client`](link/to/repo/or/docs) library.